Towny-master/plugin.yml

```diff
@@ -2,7 +2,7 @@ name: Towny
 main: Happysttim\Towny\Main
 version: Beta_0.0.1
 author: Happysttim
-api: [1.13.0,2.0.0]
+api: [2.0.0, 3.0.0]

```

Comment:
Actually, we're still in the ALPHA stages. That said, the latest API is API 3.0.0-ALPHA5